airgun/views/common.py

```diff
@@ -323,6 +324,15 @@ class SearchableViewMixin(WTMixin):
     """
     searchbox = Search()
 
+    def is_searchable(self):
+        """Verify that search procedure can be executed against specific page.
+        That means all necessary search controls are present on the page
```

Comment:
So we don't check that 'welcome' message is shown on screen? Without that, if we're accidentally on wrong page we won't figure that out as search will pass but return no results. Exception about searchbox not being present sounds better to me. Thoughts?

Reply:
There are different types of welcome messages and there are pages where no messages like that present at all
But yeah, probably that logic is invalid as we need to check only field is present